fastlane/Allurefile

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+
+allure_project_id = 1
+allure_endpoint = 'https://streamio.testops.cloud'
+allure_regression_testplan = 'Regression Testing'
+allure_results_path = 'allure-results'
+
+desc 'Upload test results to Allure TestOps'
+lane :allure_upload do
+  github_path = "/repos/#{ENV['GITHUB_REPOSITORY']}/actions/runs/#{ENV['GITHUB_RUN_ID']}"
+  github_run = JSON.parse(github_api(path: github_path)[:body])
+  github_branch = github_run['head_branch']
+  tags = []
+  launch_name =
+    if github_run['event'] == 'pull_request'
+      github_path = URI(github_run['pull_requests'][0]['url']).path
+      pull_request = JSON.parse(github_api(path: github_path)[:body])
+      tags = pull_request['labels'].map { |label| { name: label['name'] } }
+      pull_request['title']
+    elsif github_run['head_commit']
+      github_run['head_commit']['message']
+    else
+      github_run['name']
+    end
+  launch_id = allure_launch(name: launch_name, tags: tags)['id']
+  allure_args = "-e #{allure_endpoint} --project-id #{allure_project_id} --launch-id #{launch_id}"
+  sh("./xcresults export test_output/StreamChatUITestsApp.xcresult #{allure_results_path}")
+  sh("env BRANCH_NAME='#{github_branch}' ./allurectl upload #{allure_args} #{allure_results_path}")
+  UI.success("Check out test results in Allure TestOps: #{ENV['ALLURE_ENDPOINT']}/launch/#{launch_id} 🎉")
+end
+
+desc 'Create test-case in Allure TestOps and get its id'
+lane :allure_testcase do
```

Comment:
let's add verbs/intent to the names of these lanes to be more specific about their function.

eg.
`allure_createTestCase`
`allure_startRegression`

Reply:
@alteral

Reply:
Done 👍